cmd/buildah/main.go

```diff
@@ -27,6 +29,9 @@ type globalFlags struct {
 	StorageOpts       []string
 	UserNSUID         []string
 	UserNSGID         []string
+	CPUProfile        string
+	cpuProfile        *os.File
+	MemoryProfile     string
```

Comment:
not a huge fan of having the same variable names in a struct with only casing as a difference, but personal twitch.

Reply:
Yes I agree, this is bad.  Can we change this to cpuProfileFD or something like that.